doc/UsersGuide/CustomizingTheWorkflow/InputOutputFiles.rst

```diff
@@ -20,8 +20,9 @@ The external model files needed for initializing an experiment can be obtained i
 ways, including: 
 
    * Pulled from the `SRW App Data Bucket <https://registry.opendata.aws/noaa-ufs-shortrangeweather/>`__,
-   * Pulled from the NOAA High Performance Storage System (:term:`HPSS`) during the workflow execution (requires user access), or 
-   * Obtained and staged by the user from a different source. 
+   * Pulled from the NOAA High Performance Storage System (:term:`HPSS`) during the workflow execution (requires user access), 
+   * Obtained and staged by the user from a different source, or 
+   * Pulled from the `NOAA-RRFS AWS S3 bucket <https://noaa-rrfs-pds.s3.amazonaws.com/index.html#rrfs_a/>`__, a description could be found in `NOAA Rapid Refresh Forecast System (RRFS) <https://registry.opendata.aws/noaa-rrfs/>`__.
```

Comment:
```suggestion
   * Pulled from the `RRFS data bucket <https://registry.opendata.aws/noaa-rrfs/>`_.
```
We're generally supposed to link only to that landing page, not directly to the data, because the landing page has all the licensing info and such. There are places where I'll link to the data, but it's usually with a very specific purpose. Here, we're just pointing them to the buckets, and we should do it in the same way for both.